### PR TITLE
fix: clean out the panel type change attribute dependency

### PR DESCRIPTION
**Pull request by vikrantgupta25**
### Summary

- clean out the `attribute to datasource to panel type map` and added the missing attributes 
- clean out the `handleQueryChange`  function as well 
- details here - https://www.notion.so/signoz/updated-panel-types-attributes-19feddcce1c545c489932546709c4dab#56fcc2f2c2784e7abdde25613a572752

#### Related Issues / PR's

fixes https://github.com/SigNoz/engineering-pod/issues/1620 
fixes https://github.com/SigNoz/signoz/issues/5644
fixes https://github.com/SigNoz/signoz/issues/5575

#### Screenshots

NA

### CODE REVIEW

File: frontend/src/container/NewWidget/utils.ts [hunk 1]
```diff
@@ -419,7 +445,6 @@ export function handleQueryChange(
 					set(tempQuery, 'offset', 0);
 					set(tempQuery, 'pageSize', 10);
 				} else if (tempQuery.aggregateOperator === 'noop') {
-					set(tempQuery, 'aggregateOperator', 'count');
```
Comment (srikanthccv) [inline frontend/src/container/NewWidget/utils.ts:422]: Why was this added and does it break anything if we remove it now?

Comment (vikrantgupta25) [reply frontend/src/container/NewWidget/utils.ts:422]: when shifting from any panel other than list type we modify the operator to `noop` for list panel to work, we do not update the superset query here. so when moving back to any other panel from list panel the older operator should be selected by default we should not update it back to `count` here as it could have been anything.

**Comment by vikrantgupta25** [comment]
- i found one issue with moving from list panel to other panel types when list panel was the first one to be opened. fixing this.

**Comment by vikrantgupta25** [comment]
> i found one issue with moving from list panel to other panel types when list panel was the first one to be opened. fixing this.

this is fixed.

**Review by srikanthccv — APPROVED** [review]

**Review by YounixM — APPROVED** [review]
